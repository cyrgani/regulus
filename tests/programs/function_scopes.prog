_(
    def(foo, x, _(
        =(y, 0),
        x
    )),

    print(foo(2)),
    print(catch(y))
)