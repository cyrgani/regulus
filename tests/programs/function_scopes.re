def(foo, x, _(
    =(y, 0),
    x
)),

assert_eq(foo(2), 2),
__builtin_print_catch(y)