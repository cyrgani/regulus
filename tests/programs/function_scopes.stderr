NameError: No variable named `y` found!
